_rules/html-lang-valid-bf051a.md

```diff
@@ -160,20 +160,4 @@ The rule applies to `html` element and hence usage of `xml:lang` attribute in `s
 
 ```svg
 <svg xmlns="http://www.w3.org/2000/svg" xml:lang="fr"></svg>
-```
-
-#### Inapplicable Example 3
```

Comment:
Why removing these inapplicable examples?

Why is this rule not updated with "not whitespace" in the applicability?